src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -8,148 +8,175 @@
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/react/16.2.0/umd/react.production.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.2.0/umd/react-dom.production.min.js"></script>
+  <!--
+    This GraphiQL example depends on Promise and fetch, which are available in
+    modern browsers, but can be "polyfilled" for older browsers.
+    GraphiQL itself depends on React DOM.
+    If you do not want to rely on a CDN, you can host these files locally or
+    include them directly in your favored resource bunder.
+  -->
+  <script
+    src="https://cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js"
+    integrity="sha384-dcF7KoWRaRpjcNbVPUFgatYgAijf8DqW6NWuqLdfB5Sb4Cdbb8iHX7bHsl9YhpKa"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://unpkg.com/react@16.14.0/umd/react.production.min.js"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://unpkg.com/react-dom@16.14.0/umd/react-dom.production.min.js"
+    crossorigin="anonymous"
+  ></script>
 
-	<!--
-	  These two files can be found in the npm module, however you may wish to
-	  copy them directly into your environment, or perhaps include them in your
-	  favored resource bundler.
-	 -->
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/graphiql/1.0.6/graphiql.min.css" />
-	<script src="https://cdn.jsdelivr.net/npm/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"></script>
-
-	<script src="https://unpkg.com/subscriptions-transport-ws@0.9.18/browser/client.js"></script>
-	<script src="https://unpkg.com/graphiql-subscriptions-fetcher@0.0.2/browser/client.js"></script>
+  <!--
+    These two files can be found in the npm module, however you may wish to
+    copy them directly into your environment, or perhaps include them in your
+    favored resource bundler.
+   -->
+  <link rel="stylesheet" href="https://unpkg.com/graphiql@1.5.1/graphiql.min.css" />
+  <script
+    src="https://unpkg.com/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"
+    integrity="sha384-TqI6gT2PjmSrnEOTvGHLad1U4Vm5VoyzMmcKK0C/PLCWTnwPyXhCJY6NYhC/tp19"
+    crossorigin="anonymous"
+  ></script>
+  <link
+    rel="stylesheet"
+    href="https://unpkg.com/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.css"
+    integrity="sha384-GBqwox+q8UtVEyBLBKloN5QDlBDsQnuoSUfMeJH1ZtDiCrrk103D7Bg/WjIvl4ya"
+    crossorigin="anonymous"
+  />
+  <script src="https://unpkg.com/subscriptions-transport-ws@0.8.2/browser/client.js"></script>
+  <script src="https://unpkg.com/graphiql-subscriptions-fetcher@0.0.2/browser/client.js"></script>
 
 </head>
 <body>
-	<div id="graphiql">Loading...</div>
-	<script>
+  <div id="graphiql">Loading...</div>
+  <script
+    src="https://unpkg.com/graphiql@1.5.1/graphiql.min.js"
+    type="application/javascript"
```

Comment:
I think that type attribute is deprecated. No?

Reply:
Actually, the type is defaulted to `text/javascript` when not provided. This stack overflow answer seems the most accurate with references to the HTML specs per version: https://stackoverflow.com/a/4195504. I wonder why the maintainer of graphiql chose `application/javascript` over `text/javascript` 🤔.
RFC 4329 indicates that `text/javascript` mime type is obsklet though: https://datatracker.ietf.org/doc/html/rfc4329#section-7.1
I guess we could remove it, assuming all recent browsers support those specs.

Reply:
I will do some tests with various browsers and revert back.

Reply:
I would remove. If one really needs support for old browsers then he can use `GraphiQLOptions.PostConfigure` to inject/ovewrite any part of html.

Reply:
I removed it, without any issue 👍